waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/server/FederatedHMSHandlerTest.java

```diff
@@ -345,17 +345,20 @@ public void get_tables() throws TException {
     when(primaryMapping.transformInboundDatabaseName(DB_P)).thenReturn("inbound");
     List<String> tables = Lists.newArrayList("table1");
     when(primaryClient.get_tables("inbound", "*")).thenReturn(tables);
+    when(databaseMappingService.filterTables(DB_P, tables, primaryMapping)).thenReturn(tables);
     List<String> result = handler.get_tables(DB_P, "*");
     assertThat(result, is(tables));
   }
 
   @Test
   public void get_all_tables() throws TException {
```

Comment:
Any idea why the method names here are snake case?

Reply:
I think because those are their names in the original HMSHandler and it's easier to just copy them.

Reply:
yes